Sources/MongoSwift/ClientSession.swift

```diff
@@ -103,7 +103,7 @@ public final class ClientSession {
         /// A transaction has been started, but no operation has been sent to the server.
         case starting
         /// A transaction is in progress.
-        case inProgress
+        case inProgress = "in_progress"
```

Comment:
The spec tests were (annoyingly) updated to use snake_case here instead of camelCase, requiring this change.